src/assets/_project/_blocks/components/link/_qg-link.scss

```diff
@@ -5,7 +5,7 @@
 //-------------------------------------------------
 
 // global link styles
-@mixin qg-global-link-styles($margin: 2px) {
+@mixin qg-global-link-styles($margin: 2px 2px 2px 0) {
```

Comment:
Reducing the last margin to 0 causes the top-of-page buttons, "Find services" and "Business and industry", to be too crowded; the outline box for Find Services gets hidden under the next button.